web/src/main.tsx

```diff
@@ -4,7 +4,7 @@ import 'react-spring-bottom-sheet/dist/style.css';
 import './index.css';
 
 import { Capacitor } from '@capacitor/core';
-import * as Sentry from '@sentry/react';
```

Comment:
Are we sure there are no side effects to changing this?

We are no longer importing the whole of Sentry.

Reply:
Hmm, good point, I've included it again!